athletics.lic

```diff
@@ -0,0 +1,223 @@
+      @end_skill=30
+    end
+
+    if @settings.hometown == 'Crossing' || @settings.hometown == 'Riverhaven'
+      crossing_athletics
+    elsif @settings.hometown == 'Shard'
+      if DRSkill.getrank('Athletics') <= 250
+        shard_athletics
+      else
+        if DRSkill.getrank('Stealth') >= 250
+          climb_practice(19460,"wall",true)
+        else
+          hib_athletics
+        end
+      end
+    elsif @settings.hometown == 'Hibarnhvidar'
+      hib_athletics
+    elsif @settings.hometown == 'Ratha'
+      ratha_athletics
+    end
+  end
+  
+  def crossing_athletics
+    if DRSkill.getrank('Athletics') >= 400
+      return if @researching
+      climb_branch
+    else
+      while DRSkill.getxp('Athletics') < @end_skill
+        @athletics_options
+          .reject { |data| @settings.avoid_athletics_in_justice && data['justice'] }
+          .each do |data|
+            break unless climb?(data['room'], data['targets'])
+        end
+      end
+    end
+  end
+  
+  def shard_athletics
+    while DRSkill.getxp('Athletics') <= @end_skill
+      walk_to(11481)
+      walk_to(11482)
+      walk_to(11483)
+    end    
+    
+    walk_to(@settings.safe_room)
+  end
+  
+  def hib_athletics
+    walk_to(4155)
+    move('nw')
+    
+    while DRSkill.getxp('Athletics') <= @end_skill
+      fput('climb bank')
+      3.times do
+        fput('south')
+        waitrt?
+      end 
+      walk_to(4155)
+      move('nw')
+    end
+    fput('climb bank')
+    waitrt?
+    walk_to(@settings.safe_room)
+  end
+  
+  def ratha_athletics
+    while DRSkill.getxp('Athletics') <= @end_skill
+      if DRSkill.getrank('Athletics') <= 185
+        climb_practice(5287, 'rock gorge',true)
+      else
+	    if DRSkill.getrank('Stealth') >= 130
+	      climb_practice(5143, 'deep crack',true)
+		end
+      end
+    end
+  end
+  
+  def climb_practice(room, target, to_hide = false)
+    return unless target
+    walk_to(room)
+    play_song?
+    while DRSkill.getxp('Athletics') <= @end_skill
+      retreat
+      if to_hide
+        attempts_to_hide = 5
+        while attempts_to_hide > 0
+          break if hide?
+          attempts_to_hide -= 1
+        end
+        return if attempts_to_hide.zero?
+      end
+      Flags.add('ct-climbing-finished', 'You finish practicing your climbing')
+      Flags.add('ct-climbing-combat', 'You are engaged in combat')
+      bput("climb practice #{target}", 'You begin to practice ')
+      loop do
+        pause 1
+        break if Flags['ct-climbing-finished']
+        return if Flags['ct-climbing-combat']
+        if DRSkill.getxp('Athletics') > @end_skill
+          fput('stop climb')
+          break
+        end
+      end
+    end
+    stop_play
+    bput('unhide', 'You come out of hiding', 'You slip out of hiding', 'But you are not')
+    walk_to(@settings.safe_room)
+  end
+  
+  def climb?(room, targets)
+    targets.each do |target|
+      walk_to(room)
+      return true if DRRoom.npcs.length >= 3
```

Comment:
This sounds like something we could do in the future but would not hold up this PR.